validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorCoordinator.java

```diff
@@ -235,6 +228,16 @@ public void onNewAssignment(ValidatorAssignmentEvent event) throws IllegalArgume
     }
   }
 
+  @Subscribe
+  public void onAggregationEvent(AggregationEvent event) {
```

Comment:
(discussion) Similar to the above comment, maybe we could rename this event to the more explicit `BroadcastAggregationEvent`?

Reply:
done